cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/ApiConnectionInfoBuilder.java

```diff
@@ -79,6 +81,11 @@ public ApiConnectionInfoBuilder withHostnameVerifier(HostnameVerifier hostnameVe
         return this;
     }
 
+    public ApiConnectionInfoBuilder withCache(Cache cache) {
```

Comment:
This is exporting the OkHttp API

Reply:
What do you suggest?  A supplier class sort of like the CookieJar uses?  Doesn't that still tie onto the OkHttp classes?